tools/combine-files/combine_files.py

```
@@ -1311,6 +1312,258 @@ def _get_relative_path_or_original(config_file, file_path_root):
+                        # diff can fail for clean files, assume clean
+                        status = "clean"
+        except (git.exc.GitCommandError, ValueError):
+            # If we can't determine status, mark as unknown rather than silently assuming clean
+            status = "unknown"
+
+        return {
+            "repo_name": repo_name,
+            "file_path_in_repo": file_path_normalized,
+            "commit": commit,
+            "status": status,
+        }
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)